app/admin/user.rb

```diff
@@ -105,13 +106,15 @@
       f.input :volunteer
       f.input :branch_of_service
       f.input :years_of_service
+      f.input :military_status, as: :select, collection: User::MILITARY_STATUSES.map { |status| [status, status].compact }.reject(&:empty?), include_blank: true
       f.input :pay_grade
       f.input :military_occupational_specialty
       f.input :github
       f.input :twitter
       f.input :linkedin
       f.input :employment_status
       f.input :education
+      f.input :military_status
```

Comment:
You'll want to delete this input (we'll favor the new collection input on line 109).